README.md

```diff
@@ -43,21 +43,19 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [📝 License](#license)
+
```

Comment:
- [ ] I noticed you deleted the license totally. This is an important part of your project and must be included in your README File. Kindly add it up and reference it properly just as it is on the Template given. 
- Also add the License file at the root of your project so it is clickable to that part